src/download.rs

```diff
@@ -3,11 +3,14 @@ use client::GetResponse;
 use hyper::client::Client;
 use hyper::error::Error;
 use hyper::header::{ByteRangeSpec, Headers, Range};
-use pbr::{MultiBar, Pipe, ProgressBar};
+use pbr::{MultiBar, Pipe, ProgressBar, Units};
 use response::CheckResponseStatus;
 use std::cmp::min;
 use std::io::Read;
 use std::thread;
+use std::time::{Instant, Duration};
+
+const DOWNLOAD_BUFFER_BYTES: usize = 1024*64;
```

Comment:
Just add a single comment to explain why you multiply a buffer size (1024) by 64, please :-)